libraries/plugins/witness/witness.cpp

```diff
@@ -74,6 +77,8 @@ void witness_plugin::plugin_set_program_options(
           DEFAULT_VALUE_VECTOR(std::make_pair(chain::public_key_type(default_priv_key.get_public_key()),
                 graphene::utilities::key_to_wif(default_priv_key))),
                 "Tuple of [PublicKey, WIF private key] (may specify multiple times)")
+         ("private-key-file", bpo::value<vector<boost::filesystem::path>>()->composing()->multitoken(),
+          "Path to file containing tuples of [PublicKey, WIF private key] (exactly one per line)")
```

Comment:
So, we can have multiple files, and in each file we can have multiple keys? I think we need to describe it in the string.

Reply:
```cpp
("private-key-file", bpo::value<vector<boost::filesystem::path>>()->composing()->multitoken(),
"Path to a file containing tuples of [PublicKey, WIF private key]."
" The file has to contain exactly one tuple (i.e. private - public key pair) per line."
" This option may be specified multiple times, thus multiple files can be provided.")
```
Will this make it?

Reply:
Yes, thanks.

Reply:
Done.

Reply:
Thanks.